crates/sui-framework/packages/dwallet-system/sources/dwallet.move

```diff
@@ -310,4 +310,27 @@ module dwallet_system::dwallet {
         };
         transfer::transfer(sign_output, session.sender);
     }
+
+    /// Event to start a `MockMPCSession`, caught by the Validators.
+    struct CreatedProofMPCSessionEvent has copy, drop {
```

Comment:
I'm not a fan of the name. Esp. the `Created` part.

Reply:
This is an event, it was `Created`, `New` is worse since it makes it look like you are calling some constructor, I asked it.

Reply:
so long it follows conventions